ec/src/models/mnt4/mod.rs

```diff
@@ -53,9 +53,10 @@ pub trait MNT4Config: 'static + Sized {
             .zip_eq(b)
             .map(|(a, b)| (a.into(), b.into()))
             .collect::<Vec<_>>();
-        let result = ark_std::cfg_into_iter!(pairs)
+        let result = ark_std::cfg_into_iter!(pairs, 1000)
```

Comment:
is there a specific reason for 1000?

Reply:
Just a number I’m trying right now. I think it can be much lower.